helpers/argumentParsers.go

```diff
@@ -107,61 +107,61 @@ func ParseOpenFlags(flags uint32) string {
 
 	// access mode
 	switch {
-	case flags&01 == 01:
+	case eventArgumentContainsOption(uint64(flags), O_WRONLY):
```

Comment:
Since we are now execting external users to use these consts (since Tracee is emitting raw values), maybe it would be better to strongly-type them and to implement the `stringer` interface on the types: https://pkg.go.dev/fmt#Stringer
This way we (and others) can just `fmt.Println(O_WRONLY)` instead of the developer having to check the const value and then type the const name as string, like seen in this code block.
Possibly we can use the stringer utility to automate this: https://pkg.go.dev/golang.org/x/tools/cmd/stringer

Reply:
In other words, I'm saying these identifier-to-string mapping could be moved to where we define the consts, instead of where we print them, and by that they would become reusable